src/components/CarouselImage/index.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { uniqueId } from '../../libs/utils';
+import { Consumer } from '../CarouselCard/context';
+import './styles.css';
+
+class Item extends Component {
+    constructor(props) {
+        super(props);
+        this.carouselImageID = uniqueId('carousel-content-id');
+        this.carouselindicatorID = uniqueId('indicator-id');
+    }
+
+    componentDidMount() {
+        const { privateRegisterChild, header } = this.props;
+        return setTimeout(() => (
+            privateRegisterChild({
+                containerID: this.carouselImageID,
+                indicatorID: this.carouselindicatorID,
+                header,
+            })), 0);
+    }
+
+    render() {
+        const { alternativeText, description, header, href, src } = this.props;
+        return (
+            <div
+                id={this.carouselImageID}
+                className="rainbow-carousel__panel"
```

Comment:
This component is named CarouselImage so rainbow-carousel-image is the className here. And all children from here begin with rainbow-carousel-image